src/static/js/pad.js

```diff
@@ -464,7 +471,7 @@ const pad = {
     const postAceInit = () => {
       padeditbar.init();
       setTimeout(() => {
-        padeditor.ace.focus();
+        if (!settings.focusOnEditor) padeditor.ace.focus();
```

Comment:
Isn't this backwards?

```suggestion
        if (settings.focusOnEditor) padeditor.ace.focus();
```